server/container_create_linux_test.go

```diff
@@ -73,3 +75,83 @@ func TestAddOCIBindsForSys(t *testing.T) {
 		t.Error("there is not a single /sys bind mount")
 	}
 }
+
+func TestAddDevicesPlatformPrivileged(t *testing.T) {
```

Comment:
@haircommander @saschagrunert I have added this unit test which checks if host devices are added for privileged containers , and have been trying to run the test with as below, but getting following errors:
```
go test -v -run TestAddDevicesPlatform 
# github.com/cri-o/cri-o/server [github.com/cri-o/cri-o/server.test]
./inspect_test.go:60:12: container.SetState undefined (type *oci.Container has no field or method SetState)
./inspect_test.go:172:12: container.SetState undefined (type *oci.Container has no field or method SetState)
FAIL    github.com/cri-o/cri-o/server [build failed]
```
What is the recommended way to run the tests? 
I tried running `sudo -E GOPATH=$GOPATH make testunit`, but seeing other unrelated errors. Does the mock interface issue needs to be addressed for that ?

Reply:
Yes, the tests do not work any more, see: https://github.com/cri-o/cri-o/pull/2730/files#r315060012